oceannavigator/__init__.py

```diff
@@ -87,8 +88,8 @@ def create_app(testing: bool = False, dataset_config_path: str = ""):
     app.git_tag = subprocess_check_output("git", "describe", "--tags", "--abbrev=0")
 
     if app.config.get("WSGI_PROFILING"):
-        if not os.path.isdir("./profiler_results"):
-            os.mkdir("./profiler_results")
+        path = Path("./profiler_results")
+        Path(path).mkdir(parents=True, exist_ok=True)
```

Comment:
You call `Path()` on path objects in a number of places. Once you create the path object you can use it without calling `Path()` again. 

``` suggestion
        path = Path("./profiler_results")
        path.mkdir(parents=True, exist_ok=True)
```